src/config.js

```diff
@@ -38,3 +38,26 @@ for (const key of searchParams.keys()) {
     features[matches[1]] = true
   }
 }
+
+export const rtcConfig = {
+  iceServers: [
+    {
+      urls: 'stun:openrelay.metered.ca:80',
+    },
+    {
+      urls: 'turn:openrelay.metered.ca:80',
+      username: 'openrelayproject',
+      credential: 'openrelayproject',
```

Comment:
Typically I wouldn't put credentials directly in the code like this, but this is effectively public information per [Open Relay's documentation](https://www.metered.ca/tools/openrelay/#-how-to-use).